src/Configuration.ts

```diff
@@ -109,6 +115,7 @@ const EnvironmentConfigurations: {[key:string]: Configuration} = {
     zakenIsLive: true,
     zakenUseSubmissions: true,
     zakenAllowDomains: ['APV', 'JZ'],
+    showSurveyCTA: true,
   },
```

Comment:
Is het handig om hem al expliciet op false te zetten in production?

Reply:
op zich wel verstandig ja, doe ik ff